lib/pages/contacts_tab/contacts_tab_view.dart

```diff
@@ -11,7 +12,12 @@ class ContactsTabView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: ContactsAppBar(contactsController),
+      appBar: PreferredSize(
+        preferredSize: Size.fromHeight(
+          ResponsiveUtils().isMobile(context) ? 120 : 96,
```

Comment:
have you check in tablet responsive ?

Reply:
Currently, we are support 2 breakpoint 
- Mobile
- Web and Tablet